ddtrace/settings/config.py

```diff
@@ -473,8 +473,6 @@ def __init__(self):
 
         self._128_bit_trace_id_enabled = asbool(os.getenv("DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED", True))
 
-        self._128_bit_trace_id_logging_enabled = asbool(os.getenv("DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED", False))
```

Comment:
We should remove this config from our docs

Reply:
Its not documented.